.github/workflows/release-master.yml

```diff
@@ -10,7 +10,6 @@ on:
       - "pyinst.py"
 concurrency:
   group: release-master
-  cancel-in-progress: true
```

Comment:
@pukkandan @bashonly do you guys remember why we initially cancelled and didn't let it queue up?

Reply:
No

Maybe we didn't want to amass a huge queue from a PR merge spree? And with how long the arm builds take, that is still sort of a concern (15 builds would be a ~5 hour queue)

Reply:
After a merge spree there would be quite a bit of downtime though. Imo its better to have a certain build for each commit, and it can stretch for however long it wants